docker/keys/server.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCss7yaMmb6t/sj
```

Comment:
Is this just for running locally over https?

Reply:
Yes these are generated from open SSL.